Update Set-SPOTenant.md (#491)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```yaml
Type: Boolean

Parameter Sets: (All)
Aliases:
Applicable: SharePoint Online
Required: False
Position: Named
Default value: True
Accept pipeline input: False
Accept wildcard characters: False
```
### -EnableSensitivityLabelforPDF

> Tenant admin can turn on support for PDFs with sensitivity labels for the following scenarios:

Reviewer: ```suggestion
Allows tenant admins to turn on support for PDFs with sensitivity labels for the following scenarios:
```

1. Applying a sensitivity label in Office on the web
2. Uploading a labeled document, and then extracting and displaying that sensitivity label
3. Search, eDiscovery, and data loss prevention
4. Auto-labeling policies and default sensitivity labels for SharePoint document libraries

The valid values are:

> True - Enables the support for PDF.
> False (default) - Disables the support for PDF.

Reviewer: ```suggestion
True - Enables support for PDFs.

False (default) - Disables support for PDFs.
```

```yaml
Type: Boolean

Parameter Sets: (All)
Aliases:
Applicable: SharePoint Online
Required: False
Position: Named
Default value: True
Accept pipeline input: False
Accept wildcard characters: False
```
### -EnableSensitivityLabelforPDF

Allows tenant admins to turn on support for PDFs with sensitivity labels for the following scenarios:

- Applying a sensitivity label in Office for the web.
- Uploading a labeled document, and then extracting and displaying that sensitivity label.
- Search, eDiscovery, and data loss prevention.
- Auto-labeling policies and default sensitivity labels for SharePoint document libraries.

The valid values are:

> True - Enables support for PDFs.
> 
> False (default) - Disables support for PDFs.

Reviewer: ```suggestion

- True - Enables support for PDFs.
- False (default) - Disables support for PDFs.
```

> - False (default) - Disables support for PDFs.

Author: I have fixed default value now in yaml.

> - False (default) - Disables support for PDFs.

Reviewer 2: It is still saying true for me

![image](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/assets/36666927/1d96fac5-7bbf-4b58-9f10-9893fafd24f8)

> - False (default) - Disables support for PDFs.

Author: My bad. Should reflect now.
